OpenRA.Mods.Common/Traits/Render/WithCrateBody.cs

```diff
@@ -23,6 +23,10 @@ class WithCrateBodyInfo : ITraitInfo, Requires<RenderSpritesInfo>
 		[Desc("Easteregg sequences to use in december.")]
 		public readonly string[] XmasImages = { };
 
+		[SequenceReference] public readonly string IdleSequence = "idle";
+		[SequenceReference] public readonly string WaterSequence;
+		[SequenceReference] public readonly string LandSequence;
```

Comment:
> Seems like you missed the defaults here?

It is redundant for crates that are not parachuted and don't need a landing sequence (for land and water) as they just appear and stay idle.